swagger2.0/oic.wk.res.swagger.json

```diff
@@ -31,7 +85,7 @@
                     "eps": [
                       {"ep": "coaps://[fe80::b1d6]:1111", "pri": 2},
                       {"ep": "coaps://[fe80::b1d6]:1122"},
-                      {"ep": "coaps+tcp://[2001:db8:a::123]:2222", "pri": 3}
+                      {"ep": "coap+tcp://[2001:db8:a::123]:2222", "pri": 3}
```

Comment:
Vertical Resources are not allowed to expose non secured Endpoints.  Change to coaps.